html.go

```diff
@@ -124,7 +124,9 @@ type HTMLReadabilityOptions struct {
 
 // HTMLReadabilityOptionsValues are the global settings used for HTMLReadability.
 // TODO: Remove this from global state.
-var HTMLReadabilityOptionsValues HTMLReadabilityOptions
+var HTMLReadabilityOptionsValues HTMLReadabilityOptions = HTMLReadabilityOptions{
```

Comment:
@remLse I believe this is a backwards incompatible change. Users of docconv might be relying on the value of this package global as it was. For your project, is there any reason you can't set this package global to this value in your own app?

Note: per the TODO here, the use of package globals here is unfortunate and in a future version would most likely be removed.

Reply:
That makes sense, I didn't consider people might be relying on the global variable. However, in this specific case, I would be surprised if anyone relied on the fact that `ReadabilityUseClasses` is initialized as an empty string. If anything, I would expect users to overwrite this value (as you suggested I do). In that case, it wouldn't matter if the default value was changed. All properties of `HTMLReadabilityOptionsValues` being initialized with their zero values seems more like a placeholder and less like a feature that needs to be preserved within major releases. Still, I understand that package global variables shouldn't be changed in minor updates, as a matter of principle.

Setting the global variable in my own code would work, but I'm a bit hesitant to do so. I think it's bad style, probably for the same reason the TODO suggests removing these options from global state entirely.

How are you planning to implement these readablility options without global state? If that can be implemented by an extension of the API, I would be happy to write it. E.g. a new function that assumes `readability == true` and accepts options which override the global variables:
```
// Convert a file to plain text.
func ConvertReability(r io.Reader, mimeType string, options bool) (*Response, error) {
```

If that doesn't make sense, I'll just use the global variables. Thank you for the response!

Reply:
@remLse I think we'd probably look at something like what I wrote in my comment in https://github.com/sajari/docconv/pull/75/files#r449309398

Except instead of a list of args like `pageRange []int, maxWords int`, we'd possibly use the options pattern.

This kind of change is most likely additive (adding new types to the package), so can probably be done in a backwards compatible way.